src/homepageExperience/containers/NodejsWizard.tsx

```diff
@@ -76,7 +76,7 @@ export class NodejsWizard extends PureComponent<null, State> {
         return <InstallDependencies />
       }
       case 3: {
-        return <CreateToken />
+        return <CreateToken wizardEventName="nodeJsWizard" />
```

Comment:
Just fixing the event for the python and node wizard

Reply:
```suggestion
        return <CreateToken wizardEventName="nodejsWizard" />
```

small, but to keep things consistent.

Reply:
✅